### PR TITLE
Add FAQ Help

### PR DESCRIPTION
**Pull request by iMattPro**
Screenshot 2024-10-18 at 3 04 25 PM

### CODE REVIEW

File: event/listener.php [hunk 1]
```diff
@@ -279,6 +280,45 @@ public function validate_pwa_options($event)
...
+			{
+				$faq_rows[] = [
+					'FAQ_QUESTION' => $this->language->lang($question),
+					'FAQ_ANSWER'   => $this->language->lang($answer),
+				];
+			}
+					
+			$this->template->assign_block_vars_array('faq_block.faq_row', $faq_rows);
+		}
+	}
```
Comment (rxu) [inline event/listener.php:320]: I feel like we're duplicating the code from the help manager here. Instead we could just inject the manager service and use it like f.e.
```php
$this->language->add_lang('webpushnotifications_faq', 'phpbb/webpushnotifications');
$questions = [
  'HELP_FAQ_WPN_WHAT_QUESTION'    => 'HELP_FAQ_WPN_WHAT_ANSWER',
  'HELP_FAQ_WPN_HOW_QUESTION'     => 'HELP_FAQ_WPN_HOW_ANSWER',
  'HELP_FAQ_WPN_IOS_QUESTION'     => 'HELP_FAQ_WPN_IOS_ANSWER',
  'HELP_FAQ_WPN_SESSION_QUESTION' => 'HELP_FAQ_WPN_SESSION_ANSWER',
  'HELP_FAQ_WPN_SUBBING_QUESTION' => 'HELP_FAQ_WPN_SUBBING_ANSWER',
  'HELP_FAQ_WPN_GENERAL_QUESTION' => 'HELP_FAQ_WPN_GENERAL_ANSWER',
];
$this->help_manager->add_block('HELP_FAQ_WPN', false, $questions);
```

Comment (iMattPro) [reply event/listener.php:320]: Yeah but I dont want to write the tests for that just to eliminate a few lines of code in here

**Review by rxu — APPROVED** [review]

**Comment by iMattPro** [comment]
@marc1706  do you think this is worth adding to the core too?

**Comment by marc1706** [comment]
> @marc1706 do you think this is worth adding to the core too?

@iMattPro yeah, I think it would be nice to have this in the core as well. Can you create a PR for that?

**Pull request merged**